src/cephadm/cephadm.py

```diff
@@ -4692,6 +4693,35 @@ def command_list_images(ctx: CephadmContext) -> None:
     # print default images
     cp_obj.write(sys.stdout)
 
+
+def update_service_for_daemon(ctx: CephadmContext, daemon_list: list) -> None:
+    """ Update the unit.meta file of daemon with required service name for valid daemons"""
+
+    data = {'service_name': ctx.service_name}
+    update_daemon = ctx.name.split(',')
+    available_daemons = [d['name'] for d in daemon_list]
+    # check if all the daemon names are invalid
```

Comment:
are we checking if they're all valid or invalid?

Reply:
nevermind, hadn't looked close enough at the following check when I wrote this comment